src/CommandingOfficers/Strong.java

```diff
@@ -198,15 +193,15 @@ protected void perform(MapMaster gameMap)
         upm.addProductionPair(TerrainType.CITY, model);
         upm.addProductionPair(TerrainType.HEADQUARTERS, model);
       }
-      myCommander.addCOModifier(upm);
+      modList.add(upm);
 
       // Grant a global +2 movement buff.
       COMovementModifier moveMod = new COMovementModifier(2);
       for( UnitModel model : myCommander.unitModels )
       {
         moveMod.addApplicableUnitModel(model);
       }
-      myCommander.addCOModifier(moveMod);
+      modList.add(moveMod);
 
       // Lastly, all troops are refreshed and able to move again.
```

Comment:
This ending up outside `perform()` was an oversight. I am fixing it in the next branch (and can pull the fix forward if we decide against that branch, I guess)